<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>help_window</class>
    <widget class="QDialog" name="help_window">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>768</width>
                <height>549</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>ACG Model Cleanup - Instructions</string>
        </property>
        <property name="windowIcon">
            <iconset resource="../../ACGModelClean/resources.qrc">
                <normaloff>:/ACG.ico</normaloff>:/ACG.ico
            </iconset>
        </property>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>101</width>
                    <height>71</height>
                </rect>
            </property>
            <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/acg_logo.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>110</x>
                    <y>20</y>
                    <width>641</width>
                    <height>80</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QLabel" name="le_appname">
                        <property name="font">
                            <font>
                                <pointsize>12</pointsize>
                                <weight>75</weight>
                                <bold>true</bold>
                            </font>
                        </property>
                        <property name="text">
                            <string>TextLabel</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_2">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>110</y>
                    <width>741</width>
                    <height>421</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                    <widget class="QLabel" name="le_instructions">
                        <property name="text">
                            <string>TextLabel</string>
                        </property>
                        <property name="alignment">
                            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources>
        <include location="../../ACGModelClean/resources.qrc"/>
    </resources>
    <connections/>
</ui>
